lib/src/rules/prefer_const_constructors_in_immutables.dart

```diff
@@ -130,18 +130,6 @@ class _Visitor extends SimpleAstVisitor<void> {
 
   bool _hasMixin(ClassElement clazz) => clazz.mixins.isNotEmpty;
 
-  bool _hasOnlyConstExpressionsInIntializerList(ConstructorDeclaration node) {
-    bool hasConstError;
-
-    // put a fake const keyword and check if there's const error
-    node.constKeyword = new KeywordToken(Keyword.CONST, node.offset);
-    try {
-      hasConstError = hasErrorWithConstantVerifier(context, node);
-    } finally {
-      // restore const keyword
-      node.constKeyword = null;
-    }
-
-    return !hasConstError;
-  }
+  bool _hasOnlyConstExpressionsInIntializerList(ConstructorDeclaration node) =>
```

Comment:
I would inline this method - it does not have much functionality.

Reply:
Aha, yeah.  No reason to have it anymore...  Thanks!